dev/com.ibm.ws.session.cache_fat_config_infinispan/fat/src/com/ibm/ws/session/cache/config/fat/infinispan/SessionCacheConfigUpdateTest.java

```diff
@@ -153,11 +162,16 @@ public void testScheduleInvalidation() throws Exception {
         httpSessionCache.setScheduleInvalidationSecondHour(Integer.toString(hour2));
         httpSessionCache.setWriteFrequency("TIME_BASED_WRITE");
         httpSessionCache.setWriteInterval("15s");
-        server.setMarkToEndOfLog();
+        server.setMarkToEndOfLog(); // Only marks messages.log, does not mark the trace file
+        server.setTraceMarkToEndOfDefaultTrace();
         server.updateServerConfiguration(config);
         server.waitForConfigUpdateInLogUsingMark(APP_NAMES, EMPTY_RECYCLE_LIST);
+        String messageToCheckFor = "doScheduledInvalidation scheduled hours are " + Integer.toString(hour1) + " and " + Integer.toString(hour2);
 
-        TimeUnit.SECONDS.sleep(10); // Due to invalidation thread delay
+        // Monitor trace.log and wait for the message "doScheduledInvalidation scheduled hours are X and Y" before the test should continue
+        // This replaces the TimeUnit.SECONDS.sleep(10) used before
+        assertNotNull("Could not find Infinispan message \"" + messageToCheckFor + "\" Has the Infinispan message changed?",
```

Comment:
Please see my previous comment and apply here as well.

Reply:
Ditto!